### PR TITLE
Update Windows instructions

### PR DESCRIPTION
**Pull request by svick**
- fix Windows instructions so that they work
- remove dependency on NuGet
- other small improvements

### CODE REVIEW

File: Documentation/building/windows-instructions.md [hunk 1]
```diff
@@ -123,82 +123,62 @@ It's also possible to add /t:rebuild to build.cmd to force it to delete the prev
...
-	C:\coreclr-demo>cd runtime
-	C:\coreclr-demo\runtime>CoreRun.exe HelloWorld.exe
+```bat
+C:\coreclr-demo>cd runtime
+C:\coreclr-demo\runtime>CoreRun.exe hello.exe
+```
+
+If `CoreRun.exe` fails for some reason, you will see an empty output. To diagnose the issue, you can use `/v` to switch verbose mode on: `CoreRun.exe /v hello.exe`.
 
 Over time, this process will get easier. Thanks for trying out CoreCLR. Feel free to try a more interesting demo.
```
Comment (jkotas) [inline Documentation/building/windows-instructions.md]: I doubt that this manual process get easier. Would it make sense to link to the nuget-based getting started instead (http://dotnet.github.io/getting-started/)?

**Comment by jkotas** [comment]
👍 @svick Thank you for updating the instructions! I have been using a similar manual process w/o nuget in certain situations - it is nice to have it written down.

File: Documentation/building/windows-instructions.md (hunk 1 above)
Comment (svick) [inline Documentation/building/windows-instructions.md]: Not sure. This is the "Build CoreCLR" documentation. Presumably, if someone got here, they actually want to build it themselves.

**Comment by ellismg** [comment]
Yes, thank you very much @svick !

**Pull request merged**